### ISSUE TITLE
Bone marrow edema

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10318]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: 36155729 In 1988 an evaluation study - 10 patients with severe hip and knee pain - found out a signal alteration in fluid-sensitive sequences on MRI, like T2-weighted, and low signal intensity on T1-weighted. The pain disappeared without recurrence
after 13-36 months. The authors named that evidence “bone marrow edema” assuming that was correlated to an increased fluid component in the bone marrow. That was confirmed by the scintigraphic evidence of hyperemia and increased bone mineral metabolism. Because of the spontaneous resolution they named this condition “the transient bone marrow edema syndrome” 

PMID: 28639852 The lesions are associated with an intraosseous edema of the  subchondral bone. A bone bruise is probably the initial stage and subchondral trabecular microfractures might be the morphological correlate of the bone marrow edema.

PMID: 34109940 Fig. 3-A T2-weighted fat-saturated coronal TSE (turbo-spin-echo) image (TR [repetition time] = 3,600 ms, and TE [echo time] = 59 ms) demonstrating cystic changes and  marrow edema of the parent bone

**Parent term (use hpo.jax.org/app)**
Edema HP:0000969

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
PMID: 36155729 The Bone Marrow Edema Syndromes (BMEs) are a group of metabolic bone diseases characterized by the bone edema and a peculiar clinical aspect in which pain dominates with others related symptoms and a generally favorable prognosis.
BMES include Transient Regional Osteoporosis (OTR), which are Transient Osteoporosis of the Hip (TOH) and Regional Migratory Osteoporosis (RMO), considered by some Authors to be the same clinical entity in the isolated and multifocal form.

Also : Osteochondritis dissecans ORPHA:2764

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6000917
- label: Bone marrow edema
- parents: [Abnormal skeletal morphology (HP:0011842)](https://hpo.jax.org/app/browse/term/HP:0011842) | [Edema (HP:0000969)](https://hpo.jax.org/app/browse/term/HP:0000969)
- pmid: PMID:35522786
- def: Accumulation of excess fluid in the bone marrow.
- comment: This condition is typically detected using magnetic resonance imaging, where it appears as an area of increased signal intensity on fluid-sensitive sequences.
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10318
- synonyms:
